addons/product/models/product_template.py

```diff
@@ -283,6 +292,14 @@ def _compute_is_product_variant(self):
     def _compute_barcode(self):
         self._compute_template_field_from_variant_field('barcode')
 
+    def _compute_show_description_ecommerce(self):
+        for product in self:
+            product.show_description_ecommerce = self._get_show_description_ecommerce(product)
+
+    @api.model
+    def _get_show_description_ecommerce(self, product):
+        return False
```

Comment:
```suggestion
    def _get_show_description_ecommerce(self):
        self.ensure_one()
        return False
```
why an api.model method ?